dev/scripts/src/alluxio.org/build/cmd/build.go

```diff
@@ -36,12 +38,36 @@ var webUiDirs = []string{
 }
 
 func TarballF(args []string) error {
+	cmd := flag.NewFlagSet(Tarball, flag.ExitOnError)
 	// parse flags
-	opts, err := parseTarballFlags(args)
+	opts, err := parseTarballFlags(cmd, args)
 	if err != nil {
 		return stacktrace.Propagate(err, "error parsing build flags")
 	}
+	alluxioVersion, err := alluxioVersionFromPom()
+	if err != nil {
+		return stacktrace.Propagate(err, "error parsing version string")
+	}
+	if opts.artifactOutput != "" {
+		artifact, err := artifact.NewArtifact(
```

Comment:
```suggestion
		a, err := artifact.NewArtifact(
```
masking the `artifact` package import